products/sle/main.pm

```diff
@@ -1027,6 +1027,39 @@ sub load_slenkins_tests {
     return 0;
 }
 
+sub load_hpc_cluster_tests {
+    # We need to boot *before* waiting for support server
+    # Because of TianoCore UEFI boot method
+    loadtest "boot/boot_to_desktop";
```

Comment:
I think you need switch lines . first you waiting for support server to start and only after this you booting , otherwise DHCP server can't do his job

Reply:
sorry miss your comment about TianoCore UEFI ... But still my comment is valid if you doing this in this way you will not get IP from support server DHCP because your are waiting for support server AFTER your machine is booted. Need to take a look how other guys solving this...

Reply:
It would break booting as it is in comment. Was already discussed with @ldevulder in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4229

Reply:
I'm aware of the situation and it causes only problem at the beginning for qam patch_and_reboot.
I already did what you suggested, but it broke HA tests on arm. If you do other stuff, like console setup, there is enough time to get DHCP lease.  My plan is put somewhere check/wait for the network (didn't decide yet, as it is no issue at the moment).

Reply:
I review approach taken by @ldevulder for HA . You missing important peace here . So flow how it happens in HA and why it works for them :
1. boot_to_desktop 
2. wait_support_server 
3. reboot machine to get IP 
and we missing here last step . So my suggestion is :
1. split flow for two arches
2. for x86_64 do in a right way ( wait support server and then boot to desktop ) 
3. for aarch64 do as you wrote but also add rebooting step to really acquire IP 

@ldevulder what do you think ?

Reply:
Could be a good workaround, but at the end as @czerw said we may need to find a better way to handle this issue. At the end HA tests will execute on x86_64, aarch64, ppc64le and s390x, so using the same "wait for support server" could be easier :)

Reply:
I guess if it's only about dhcp, we can call it explicitly to configure network in separate module after boot.